### DISCUSSION TITLE
disregard i was using a diffrent version

### DISCUSSION THREAD

**Discussion by M-Dot613** [roleoroleo/yi-hack-Allwinner-v2#914] [Q&A]
So... i don't remember setting up a password anyway to reset it?

**Comment by roleoroleo** [comment]
If you mean password for http access, you can access the cam with ssh (it's a different password) and find the http password in /tmp/sd/yi-hack/etc/system.conf.
If you mean root password, you can access the cam with a browser and change it in the Configuration page.
Or you can remove the sd card and use a PC to read it.
